frontend/src/Map/Map.tsx

```diff
@@ -48,7 +63,7 @@ function Map({ mapContainer, murals }: IMapProps) {
     });
 
     murals.forEach((mural: any) => {
-      new mapboxgl.Marker()
+      new mapboxgl.Marker({ color: "#FC323E" })
```

Comment:
Maybe parametrize this or put in constants? Not big deal

Reply:
I realized this issue might get closed by Blanche's next PR so I'll wait a bit to merge